src/api/spec/lib/influx_db/obs/middleware/backend_subscriber_spec.rb

```diff
@@ -21,9 +21,8 @@
 
     let(:result) do
       {
-        values: {
-          value: 2000
-        },
+        # :request might be nil. In that case we have to filter it out.
+        values: InfluxDB::Rails.current.values.reject { |_, v| v.nil? }.update(value: 2000),
```

Comment:
why do you need to reject things? :confused:

Reply:
[Apparently](https://circleci.com/gh/bgeuken/open-build-service/1749#tests/containers/2) there can be a nil value in the request. And the backend subscriber [filters](https://github.com/openSUSE/open-build-service/blob/master/src/api/lib/influxdb_obs/obs/middleware/backend_subscriber.rb#L38) nil values.

Reply:
why do you need to do it as well then?

Reply:
Because this hash is used in the [test](https://github.com/openSUSE/open-build-service/blob/master/src/api/spec/lib/influx_db/obs/middleware/backend_subscriber_spec.rb#L48) as expected parameter when `write_point` is called.